gufe/components/proteincomponent.py

```diff
@@ -346,7 +353,7 @@ def _from_dict(cls, ser_dict: dict, name: str = ""):
         for bond in ser_dict["bonds"]:
             atomBeginIdx = bond[0]
             atomEndIdx = bond[1]
-            bondType = bond[2]
+            bondType = BondType.values[bond[2]]
```

Comment:
If we're worried about rdkit breaking their enums we could do have our own equivalent enum for serialisation, but I'd be surprised if they broke this.

Reply:
This was my thought, too. I think the risk is very small, but it is nonzero. It's more likely that rdkit might expand their enums with new items, and this way we'd automatically have those new items be available. But maybe that doesn't matter for our use cases.

Anyway, I'm fine with it either way. Let me know which you prefer.